packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -155,3 +159,110 @@ export interface AppWithRepository extends Application {
   component(component: Class<{}>): void;
   mountComponentRepository(component: Class<{}>): void;
 }
+
+/**
+ * A dummy class created to generate the tsdoc for the members in repository
+ * mixin. Please don't use it.
+ *
+ * The members are implemented in function
+ * <a href="#RepositoryMixin">RepositoryMixin</a>
+ */
+export class RepositoryMixinDoc {
+  // A mixin class has to take in a type any[] argument!
+  // tslint:disable-next-line:no-any
+  constructor(...args: any[]) {}
```

Comment:
We can mark the class abstract or throw an error in its constructor to avoid misuse.

Reply:
added an error in the constructor function. 👍